### PR TITLE
[New] Manually add jQuery.scrollSpeed v1.0~v1.0.2

### LINKED ISSUE #6621: [Request] Add Scrollspeed
**Library name: Scrollspeed** 
**Git repository url: https://github.com/nathco/jQuery.scrollSpeed**
**npm package url(optional):** 
**License(s):**
**Official homepage:** https://code.nath.co/scrollSpeed
**Wanna say something? Leave message here:**
# 

Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by sashberd**
close #6621, cc @nathco, using web-minify-helper for files minifing. The files were added from release notes.

Pull request for issue: #6621 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/nathco/jQuery.scrollSpeed
 * Official website (optional, not the repository): https://code.nath.co/scrollSpeed
 * NPM package url (optional):
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 14
   - Count of stars: 290
   - Count of forks: 76
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/jquery.scrollspeed/package.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "jquery.scrollspeed",
+  "filename": "jQuery.scrollSpeed.min.js",
+  "description": "Extension for custom scrolling speed and easing",
+  "version": "1.0.2",
+  "author": "Nathan Rutzky",
+  "homepage": "https://code.nath.co/scrollSpeed",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/brightcove/videojs-thumbnails.git"
```
Comment (sufuf3) [inline ajax/libs/jquery.scrollspeed/package.json]: The repo's URL is not the current lib which be added by you.
Please modify this.
Thanks

File: ajax/libs/jquery.scrollspeed/package.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  },
+  "keywords": [
+    "jquery",
+    "scrolling",
+    "easing"
+  ],
+  "license": "MIT"
```
Comment (sufuf3) [inline ajax/libs/jquery.scrollspeed/package.json]: How did you know this lib is under MIT license?
Is there any reference?
Thanks

**Comment by sashberd** [comment]
Hi @nathco,
We would like to host you library on cdnjs, however, we did not find any reference for license of your library. Could you help us and tell what license is? (I think it is MIT, but no reference was found 😄 )
Thanks for you coopeeration

**Comment by sashberd** [comment]
@sufuf3 The PR was updated
Lets wait for author answer here, no issues tab in project. If the license is MIT and PR is LGTM after current update we could merge it

**Comment by sashberd** [comment]
f4374d6ba00b02936e8cffea848901b4eee04000

**Comment by sashberd** [comment]
[github.com/nathco/jQuery.scrollSpeed@f4374d6ba00b02936e8cffea848901b4eee04000](url)

**Comment by sashberd** [comment]
@sufuf3 The PR was updated. FYR

**Comment by sufuf3** [comment]
@sashberd 
The lib's name in commit message also need to be revised.
Could you please modify it?
Many thanks.

**Comment by sashberd** [comment]
@sufuf3 Done

**Comment by sufuf3** [comment]
@sashberd Would you please try to modify this PR first? This PR is created by you.

**Comment by sashberd** [comment]
@sufuf3 I updated commit message. FYR

**Comment by sashberd** [comment]
@PeterDaveHello The version 1.0.2 was grabbed from PR you provided. FYR

**Pull request merged**